backend/sozisel/lib/sozisel/model/participants/participant.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Sozisel.Model.Participants.Participant do
+  use Sozisel.Model.Schema
+  import Ecto.Changeset
+
+  alias Sozisel.Model.Participants
+  alias Sozisel.Model.Events.EventResult
+
+  @type t :: %__MODULE__{
+          id: Ecto.UUID.t(),
+          full_name: String.t(),
+          email: String.t(),
+          token_id: String.t(),
+          inserted_at: DateTime.t(),
+          updated_at: DateTime.t()
+        }
+
+  schema "participants" do
```

Comment:
If we have participants table then we can simply use `participant_id` in `event_results` table, am I right?

Change `token_id` to `token`.